CS3230Project/View/EditPatient.cs

```diff
@@ -10,38 +11,71 @@ namespace CS3230Project.View
     /// </summary>
     public partial class EditPatient : Form
     {
+        private Dictionary<string, string> updatedDetails;
+        private string[] states;
+
         /// <summary>
         /// Initializes a new instance of the <see cref="EditPatient" /> class.
         ///s </summary>
         /// <param name="patient">The patient.</param>
         public EditPatient(Patient patient)
         {
             this.InitializeComponent();
+            this.updatedDetails = new Dictionary<string, string>();
             this.bindLabelsToCurrentUser();
             this.setStatusComboBoxValues();
+            this.setGenderComboBoxValues();
             this.loadPatientData(patient);
+            this.updatedDetails.Add("PatientId", patient.PatientId.ToString());
+
+            this.states = new[]
```

Comment:
The dropdowns should have an items property where you can input the states. For some reason the items are not showing for the register patient on your branch but they are showing on master. It should not be an issue but just be aware when merging the branches.

![Capture](https://user-images.githubusercontent.com/23218115/193928714-f9d9fd0c-88cb-4221-b84b-931ac3bd8644.PNG)

Reply:
I have removed from the code, and added it through the editor. I will check the Register patient once merged.